vars/newtestpipeline.groovy

```diff
@@ -0,0 +1,160 @@
+                                    #!/bin/bash
+                                    set -xe
+                                    env
+
+                                    export SCT_CLUSTER_BACKEND="${params.backend}"
+                                    export SCT_REGION_NAME=${aws_region}
+                                    export SCT_CONFIG_FILES="${test_config}"
+                                    export SCT_COLLECT_LOGS=false
+
+                                    if [[ ! -z "${params.scylla_ami_id}" ]] ; then
+                                        export SCT_AMI_ID_DB_SCYLLA="${params.scylla_ami_id}"
+                                    elif [[ ! -z "${params.scylla_version}" ]] ; then
+                                        export SCT_SCYLLA_VERSION="${params.scylla_version}"
+                                    elif [[ ! -z "${params.scylla_repo}" ]] ; then
+                                        export SCT_SCYLLA_REPO="${params.scylla_repo}"
+                                    else
+                                        echo "need to choose one of SCT_AMI_ID_DB_SCYLLA | SCT_SCYLLA_VERSION | SCT_SCYLLA_REPO"
+                                        exit 1
+                                    fi
+
+                                    export SCT_FAILURE_POST_BEHAVIOR="${pipelineParams.params.get('post_behaviour', '')}"
+                                    export SCT_INSTANCE_PROVISION="${pipelineParams.params.get('provision_type', '')}"
+                                    export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$GIT_BRANCH | sed -E 's+(origin/|origin/branch-)++')
+                                    export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$SCT_AMI_ID_DB_SCYLLA_DESC | tr ._ - | cut -c1-8 )
+
+                                    echo "start test ......."
+                                    ./docker/env/hydra.sh run-test ${pipelineParams.test_name} --backend ${params.backend}  --logdir /sct
+                                    echo "end test ....."
+                                    """
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            stage('Collect log data') {
+                steps {
+                    catchError(stageResult: 'FAILURE') {
+                        script {
+                            wrap([$class: 'BuildUser']) {
+                                dir('scylla-cluster-tests') {
+                                    def aws_region = groovy.json.JsonOutput.toJson(params.aws_region)
+                                    def test_config = groovy.json.JsonOutput.toJson(pipelineParams.test_config)
+
+                                    sh """
+                                    #!/bin/bash
+
+                                    set -xe
+                                    env
+
+                                    export SCT_CLUSTER_BACKEND="${params.backend}"
```

Comment:
since those are command we want to be able to run manually, please try avoid using the environment variable and pass everything need via the command line arguments

Reply:
for example I don't think the configuration file is needed here

Reply:
adding explicit parameters for command clean-resources, collect-logs.